src/profile_manager/views.py

```diff
@@ -59,19 +77,36 @@ class ProfileListCurrent(ProfileList):
     Arguments:
         ProfileList -- Base class
     """
+    view_type = ViewTypes.CURRENT
 
     # override the staff requirement for ProfileList
     def test_func(self):
         return self.request.user.is_authenticated
 
     def get_queryset(self):
-        profiles_qs = Profile.objects.all_for_active_semester()
+        profiles_qs = Profile.objects.all_for_active_semester().filter(user__is_active=True)
```

Comment:
I think this is already the case?  did it show inactive users before you added the filter? I don't think so.

Reply:
despite being set as inactive, test_user still shows up in current semester
![image](https://user-images.githubusercontent.com/39788517/183835065-aff57bf9-2554-417e-9194-559bac8b4cd9.png)
![image](https://user-images.githubusercontent.com/39788517/183835178-2a319551-c9fd-440c-a82c-dcebf1bf4c01.png)
![image](https://user-images.githubusercontent.com/39788517/183835201-d1245b05-26b9-4792-b0bc-9c9e79fcb4dc.png)

Reply:
Dang it!  This needs to be fixed in all_for_active_semester(), it should already be doing that filtering. It should NOT include inactive students.

Can you also add a test to the all_for_active_semester() method that verifies inactive students are not included? Thanks!  And nice catch.

Reply:
added test_managers.py